src/layouts/VerticalLayout.tsx

```diff
@@ -24,20 +24,13 @@ export function VerticalLayout({
   }
   const form = Form.useFormInstance()
   return (
-    <Form {...getFormLayoutOptions(form)} form={form}>
-      {!groupLayout.label && (
```

Comment:
this seems like a mistake; I don't think we have a good reason for treating a VerticalLayout uischema as a GroupLayout uischema. I think you should remove this label code (lines 28-30 in new file) altogether.

Reply:
done